### DISCUSSION TITLE
Please tell me your favorite Doraemon's Secret Tools name

### DISCUSSION THREAD

**Discussion by KDR121** [NAIST-SE/AttractingContributors#115] [General]
Hi. I'm Tomohito Kodera
Please tell me your favorite Doraemon's Secret Tools name.

I'm looking for collaborators.

this is my git [link](https://github.com/KDR121/AttractingContributors).

**Comment by 9002taicher** [comment]
I added "Toriyose-bag" to yours.
Plz check my request, thx.
